src/SourceBuild/patches/xdt/0001-Allow-override-of-build-tool-framework-version.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nikola Milosavljevic <nikolam@microsoft.com>
+Date: Mon, 30 Oct 2023 22:52:31 +0000
+Subject: [PATCH] Allow override of build tool framework version
+
+Backport: https://github.com/dotnet/source-build/issues/3171
```

Comment:
I think the Backport link should reference the repo consolidation issue.
```suggestion
Backport: https://github.com/dotnet/aspnetcore/issues/51753
```

Reply:
Oh, yeah - updating - thanks!